pkg/artifactory/resource_artifactory_ldap_group_setting_test.go

```diff
@@ -0,0 +1,97 @@
+package artifactory
+
+import (
+	"encoding/xml"
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const LdapGroupSettingTemplateFull = `
+resource "artifactory_ldap_group_setting" "ldapgrouptest" {
+	name = "ldapgrouptest"
+	enabled_ldap = "ldaptest"
+	group_base_dn = "ldaptest_dn"
+	group_name_attribute = "cn"
+	group_member_attribute = "uniqueMember"
+	sub_tree = true
+	filter = "(objectClass=groupOfNames)"
+	description_attribute = "description"
+	strategy = "STATIC"
+}`
+
+const LdapGroupSettingTemplateUpdate = `
```

Comment:
Please put these constant strings in with their corresponding test cases, unless they need to be shared